From dce6520a5c9235dd9d07389f644d1872d1ba76d6 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v3-3.9-dce6520a5c9-20210911T092732Z-avarab@gmail.com>
In-Reply-To: <cover-v3-0.9-00000000000-20210911T092732Z-avarab@gmail.com>
References: <cover-v2-0.8-00000000000-20210910T105523Z-avarab@gmail.com>
	<cover-v3-0.9-00000000000-20210911T092732Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 10 Sep 2021 12:21:31 +0200
Subject: [PATCH v3 3/9] INSTALL: mention that we need libcurl 7.19.4 or newer
 to build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Without NO_CURL=Y we require at least version "7.19.4" of libcurl, see
644de29e220 (http: drop support for curl < 7.19.4, 2021-07-30). Let's
document this in the "INSTALL" document.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 INSTALL | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/INSTALL b/INSTALL
index d593f628749..795b02b6f96 100644
--- a/INSTALL
+++ b/INSTALL
@@ -144,6 +144,10 @@ Issues of note:
 	  not need that functionality, use NO_CURL to build without
 	  it.
 
+	  Git requires version "7.19.4" or later of "libcurl", to
+	  build (without NO_CURL). This version requirement may be
+	  bumped in the future.
+
 	- "expat" library; git-http-push uses it for remote lock
 	  management over DAV.  Similar to "curl" above, this is optional
 	  (with NO_EXPAT).
-- 
2.33.0.984.gea2c3555113

